packages/backend/src/index.ts

```diff
@@ -7,6 +8,12 @@ import { errorMiddleware } from './lib/apiHelpers';
 
 const app = express();
 
+app.locals.limiter = new Bottleneck({
+  maxConcurrent: 10,
```

Comment:
I was shooting for something pretty small so as to avoid any possibility of the load being an issue. It takes about five minutes to process all 500 users in the database, and most of the time only a small percentage of those will be in the caching queue.